xml/System.ServiceModel.Syndication/SyndicationFeed.xml

```diff
@@ -1982,9 +2040,18 @@
         <ReturnType>System.ServiceModel.Syndication.SyndicationTextInput</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the TextInput property for the feed.</summary>
+        <value>A <see cref="T:System.ServiceModel.Syndication.SyndicationTextInput" /> instance that contains required parameters for the RSS <c>textInput</c> item for the feed.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The RSS specification allows for a `textInput` item to be included within a syndication feed that the publisher can use
+ to direct search requests or obtain feedback through a simple text input box and button. See <xref:System.ServiceModel.Syndication.SyndicationFeed.SyndicationTextInput%2A>
```

Comment:
This xref is causing a [build warning](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C9%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202001092214452236-3743%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=qdZzs9T4%2Fw5ebt64SCqloWjFiSS092gaR5JQPDfLGao%3D&st=2020-01-09T22%3A47%3A36Z&se=2020-02-09T22%3A52%3A36Z&sp=r) (scroll to the end).
The cref has an extra word and an unexpected `%2A`.
It should be:

```suggestion
 to direct search requests or obtain feedback through a simple text input box and button. See <xref:System.ServiceModel.Syndication.SyndicationTextInput>
```